### ISSUE TITLE
Split data to train and validation sets, report epoch accuracy score

### ISSUE THREAD

**Issue by gwiener** [gwiener/AppRecSys#4]
Epoch score is not intuitive enough for proper hyper-parameter search. Add accuracy measures of a validation set.
